doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note*: This function does not work on AIX versions older than 7.1. It can still
+be called but will return `UV_ENOSYS`.
+
```

Comment:
Hmm, let's see how it looks with `js` formatting:

```js
Note: This function does not work on AIX versions before 7.1,
it will return the error UV_ENOSYS.
```

Yep, looks good to me.